persistence/src/commonMain/db_user/com/wire/kalium/persistence/Members.sq

```diff
@@ -17,6 +17,9 @@ VALUES (?, ?, ?);
 deleteMember:
 DELETE FROM Member WHERE conversation = ? AND user = ?;
 
+deleteMembersFromConversation:
+DELETE FROM Member WHERE conversation = ?;
+
```

Comment:
we can have a sql transaction that add members and remove the once that are not in the new list and it will be fast af

updateFullMemberList {
-- loop to insert members 

DELETE FROM Member WHERE 
conversationId = :conversationId AND userId IS NOT IN :fullMembersList 
}